sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/CreateNewContactSteps.java

```diff
@@ -19,36 +19,165 @@
+          fillPrimaryEmailAddress(contact.getPrimaryEmailAddress());
+          selectReturningTraveler(contact.getReturningTraveler());
+          fillDateOfReport(contact.getReportDate());
+          fillDiseaseOfSourceCase(contact.getDiseaseOfSourceCase());
+          fillCaseIdInExternalSystem(contact.getCaseIdInExternalSystem());
+          fillDateOfLastContact(contact.getDateOfLastContact());
+          fillCaseOrEventInformation(contact.getCaseOrEventInformation());
+          selectResponsibleRegion(contact.getResponsibleRegion());
+          selectResponsibleDistrict(contact.getResponsibleDistrict());
+          selectResponsibleCommunity(contact.getResponsibleCommunity());
+          selectTypeOfContact("any");
+          fillAdditionalInformationOnTheTypeOfContact(
+              contact.getAdditionalInformationOnContactType());
+          selectContactCategory(contact.getContactCategory());
+          fillRelationshipWithCase(contact.getRelationshipWithCase());
+          fillDescriptionOfHowContactTookPlace(contact.getDescriptionOfHowContactTookPlace());
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(UUID);
         });
   }
+
+  public void fillFirstName(String firstName) {
+    webDriverHelpers.fillInWebElement(FIRST_NAME_OF_CONTACT_PERSON_INPUT, firstName);
+  }
+
+  public void fillLastName(String lastName) {
+    webDriverHelpers.fillInWebElement(LAST_NAME_OF_CONTACT_PERSON_INPUT, lastName);
+  }
+
+  public void fillDateOfBirth(LocalDate localDate) {
+    webDriverHelpers.selectFromCombobox(
+        DATE_OF_BIRTH_YEAR_COMBOBOX, String.valueOf(localDate.getYear()));
+    webDriverHelpers.selectFromCombobox(
+        DATE_OF_BIRTH_MONTH_COMBOBOX,
+        localDate.getMonth().getDisplayName(TextStyle.FULL, Locale.ENGLISH));
+    webDriverHelpers.selectFromCombobox(
+        DATE_OF_BIRTH_DAY_COMBOBOX, String.valueOf(localDate.getDayOfMonth()));
+  }
+
+  public void selectSex(String sex) {
+    webDriverHelpers.selectFromCombobox(SEX_COMBOBOX, sex);
+  }
+
+  public void fillNationalHealthId(String nationalHealthId) {
+    webDriverHelpers.fillInWebElement(NATIONAL_HEALTH_ID_INPUT, nationalHealthId);
+  }
+
+  public void fillPassportNumber(String passportNumber) {
+    webDriverHelpers.fillInWebElement(PASSPORT_NUMBER_INPUT, passportNumber);
+  }
+
+  public void fillPrimaryPhoneNumber(String primaryPhoneNumber) {
+    webDriverHelpers.fillInWebElement(PRIMARY_PHONE_NUMBER_INPUT, primaryPhoneNumber);
+  }
+
+  public void fillPrimaryEmailAddress(String primaryPhoneNumber) {
+    webDriverHelpers.fillInWebElement(PRIMARY_EMAIL_ADDRESS_INPUT, primaryPhoneNumber);
+  }
+
+  public void selectReturningTraveler(String option) {
+    String optionBox = "//div[@id='returningTraveler']//label[contains(text(),'" + option + "')]";
+    By optionBoxXpath = By.xpath(optionBox);
+    webDriverHelpers.clickOnWebElementBySelector(optionBoxXpath);
```

Comment:
Please identify specific css selector for labels and use the `clickWebElementByText` method

Reply:
done